geoalchemy2/_functions_helpers.py

```diff
@@ -0,0 +1,83 @@
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+
+def _get_docstring(name: str, doc: Union[None, str, Tuple[str, str]], type_: Optional[type]) -> str:
+    doc_string_parts = []
+
+    if isinstance(doc, tuple):
+        doc_string_parts.append(doc[0])
+        doc_string_parts.append("see http://postgis.net/docs/{0}.html".format(doc[1]))
+    elif doc is not None:
+        doc_string_parts.append(doc)
+        doc_string_parts.append("see http://postgis.net/docs/{0}.html".format(name))
+
+    if type_ is not None:
+        return_type_str = "{0}.{1}".format(type_.__module__, type_.__name__)
+        doc_string_parts.append("Return type: :class:`{0}`.".format(return_type_str))
+
+    return "\n\n".join(doc_string_parts)
+
+
+def _replace_indent(text: str, indent: str) -> str:
+    lines = []
+    for i, line in enumerate(text.splitlines()):
+        if i == 0 or not line.strip():
+            lines.append(line)
+        else:
+            lines.append(f"{indent}{line}")
+    return "\n".join(lines)
```

Comment:
I have made the change to wrap the text. The wrapping is done before the indenting so it's wrapped to 104 columns but this can be fixed if it matters.